embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -284,10 +284,9 @@ public TaskReport run(TaskSource taskSource,
         }
 
         TaskReport report = Exec.newTaskReport();
-        // TODO
-        //if (orderByColumn != null) {
-        //    report.set("last_value", lastValue);
-        //}
+        if (task.getIncrementalColumn() != null) {
+            report.set("last_record", task.getLastRecord().get());
```

Comment:
last_record will be invalid value if this bulk data loading loaded 0 records (that may happen as a successful case).